composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "ftw-soft/intercom-notification-channel",
+    "description": "Intercom Notifications Driver",
+    "homepage": "https://github.com/ftw-soft/intercom-notification-channel",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Andrey Telesh",
+            "email": "andrey.telesh.y@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=7.1.3",
+        "guzzlehttp/guzzle": "^6.3",
+        "intercom/intercom-php": "^3.2",
+        "illuminate/notifications": ">=5.5",
+        "illuminate/support": ">=5.5"
+    },
+    "require-dev": {
+        "mockery/mockery": "^1.1",
+        "phpunit/phpunit": "~7.0",
+        "scrutinizer/ocular": "^1.5"
+    },
+    "autoload": {
+        "psr-4": {
+            "FtwSoft\\NotificationChannels\\Intercom\\": "src"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "FtwSoft\\NotificationChannels\\Intercom\\Tests\\": "tests"
```

Comment:
```suggestion
            "NotificationChannels\\Intercom\\Tests\\": "tests"
```

Reply:
I think that I should to upgrade the major version to do that

Reply:
Ahh, didn't realise it was already published. Since the package will be moving packagist namespace anyway (from `ftw-soft/intercom-notification-channel` to `laravel-notification-channels/intercom`) i don't think it's an issue.